scipy/stats/_unuran/setup.py

```diff
@@ -105,7 +105,8 @@ def configuration(parent_package="", top_path=None):
         ("STDC_HEADERS", "1"),
         ("UNUR_ENABLE_INFO", "1"),
         ("VERSION", '"%s"' % UNURAN_VERSION),
-        ("HAVE_CONFIG_H", "1")
+        ("HAVE_CONFIG_H", "1"),
+        ("_ISOC99_SOURCE", "1"),
```

Comment:
How is this different than adding a `-std=c99` flag? Is it possible that the compiler doesn't recognize the `-std=c99` flag but understands `_ISOC99_SOURCE`?

Reply:
Never mind, your commit message answers the question:

> For gcc 4.8.2 with -std=c99 _ISOC99_SOURCE is not set

Reply:
One last comment: Defining `("_ISO99_SOURCE", "1")` would read `#define _ISO99_SOURCE 1`. I think we should rather do this:

```suggestion
        ("_ISOC99_SOURCE", None),
```

which is equivalent to `#define _ISOC99_SOURCE`.

Reply:
It's usually set to `1`. For eg: in `/usr/include/features.h` it is defined as `# define _ISOC99_SOURCE 1`

Reply:
Oh, OK. Thanks for clarifying!